_output.yml

```diff
@@ -16,16 +16,18 @@ bookdown::gitbook:
 bookdown::pdf_book:
   latex_engine: "pdflatex"
   citation_package: "natbib"
+  keep_tex: yes
```

Comment:
delete this. bookdown is very fragile about where `keep_tex` should go.

Reply:
Handled by 2990f91